app/models/user.py

```diff
@@ -40,6 +41,18 @@ def isBonnerScholar(self):
     @property
     def fullName(self):
         return f"{self.firstName} {self.lastName}"
+    
+    @property
+    def getEmergencyContact(self):
+
+        from app.models.emergencyContact import EmergencyContact # Must defer import until now to avoid circular reference
+        useremergencyContact = EmergencyContact.get_or_none(user=self.username)
+        if useremergencyContact:
+            emergencyName = useremergencyContact.name or None
+            emergencyNumber = useremergencyContact.cellPhone or EmergencyContact.homePhone or EmergencyContact.workPhone or None
+        else:
+            emergencyName, emergencyNumber = None, None
+        return emergencyName, emergencyNumber
```

Comment:
Cache the values so we don't run the queries twice in one request.

Reply:
This is no longer relevant as the queries were redesigned to be done as join so that additional queries are no longer needed.